index.js

```diff
@@ -571,23 +572,31 @@ SqlPort.prototype.updateSchema = function(schema) {
 
                             var request = self.getRequest();
```

Comment:
As the functions are rewritten into arrow functions maybe use 'let' instead of 'var'. For example "var request" here will override "var request" in the begging of the "retryFailedQueries" function. This might potentially cause unexpected bugs.

Reply:
they are in different clojures

Reply:
ok ignore my comment